src/applications/vaos/containers/ClinicChoicePage.jsx

```diff
@@ -128,6 +129,7 @@ export class ClinicChoicePage extends React.Component {
             onBack={this.goBack}
             pageChangeInProgress={pageChangeInProgress}
           />
+          <NeedHelp />
```

Comment:
Could this go in NewAppointmentLayout instead?

Reply:
Good call.  I forgot we had a parent layout component.  Just pushed a commit with this change.